Issue 2604 Twenty Nineteen Backend Container Alignment

Marko64 opened the pull request:
Fixes #2604 
- Twenty Nineteen was adding a weird left value for the container. Made it auto.
- Tested if changing container position work from the transform option, and it does. Also, it works on other themes.
- Twenty Twenty Two has a similar issue. Should I also fix it here, or in a new PR?

Marko64 committed:
Issue 2604 Twenty Nineteen Backend Container Alignment

- Twenty Nineteen was adding a weird left value for the container. Made it auto.
- Tested if changing container position work from the transform option, and it does. Also it works on other themes.
- Twenty Twenty Two has a similar issue. Should I also fix it here, or in a new PR?

Naaaaiix reviewed (commented):
Good job @Marko64 🥳 

Implement the other fix if is related to theme ban with CSS please 🙏

Marko64 committed:
Twenty Twenty Two container alignment

- Twenty Twenty Two had the container aligned to the left, that's why I had to add the margin.
- It also adds container padding, so added padding: 0 for it;  5px for the wrapper exists on other themes, but not this one.

Marko64 committed:
Removed the important where possible

- Updated 2/3 CSS blocks. The first one already uses the important rule, so only way to override it is to have another important rule added. Nothing we can do here.
